cdb2api/cdb2api.h

```diff
@@ -273,8 +273,8 @@ typedef enum cdb2_event_type {
     CDB2_AT_EXIT_NEXT_RECORD = 1 << 13,
 
     /* Lifecycle events */
-    CDB2_AT_OPEN = 1 << 30,
-    CDB2_AT_CLOSE = 1 << 31,
+    CDB2_AT_OPEN = 1 << 29,
+    CDB2_AT_CLOSE = 1 << 30,
```

Comment:
?

Reply:
1 << 31 overflows the int (which is what enum is based on) and should not be used. 
We have space from 13 to 28 now.

Reply:
ICTHX